cpp/src/Slice/MD5I.cpp

```diff
@@ -172,7 +173,7 @@ md5_process(md5_state_t *pms, const md5_byte_t *data /*[64]*/)
              * On little-endian machines, we can process properly aligned
              * data without copying it.
              */
-            if (!((reinterpret_cast<uintptr_t>(data)) & 3)) {
+            if (!((data - (const md5_byte_t *)0) & 3)) {
```

Comment:
The `reinterpret_cast` is still needed here! The include is required for some compilers that can't find `uintptr_t`

Reply:
Oh, that was accidental (made a mistake with git). I'll re-apply.